types/index.d.ts

```diff
@@ -86,6 +86,20 @@ interface Dist {
   unpackedSize?: number
 }
 
+interface DevEngineDependency {
+  name: string
```

Comment:
Is there a predefined list of allowed names for certain dependency types (e.g. `os`)?

If so, do we want to enforce those via types?

Reply:
No, this is the name *of* the OS.  We don't really need to confine the spec in that way.  If a new runtime came out tomorrow called "done.js" this spec shouldn't need to update to accept it.  That's a problem for the runtime detector to worry about.

Reply:
Makes sense.

I was thinking about helping package maintainers add `devEngines` to their `package.json` files with a recognized name for `os`, `runtime`, etc. But that's a job for https://json.schemastore.org/package.json or similar, not these typings.